apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Delete_Synthetic_Negative_Claims_With_PositiveClaimIds.sql

```diff
@@ -0,0 +1,34 @@
+DELETE 
```

Comment:
Can we comment and reference the ticket ID

Reply:
sql fine as is; presonally I can't wait to change the BeneficiryID to numeric; LIKE and/or NOT LIKE are pain points for performance!